modules/spring-boot-autoconfigure/pom.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!--
+    POM file.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.ignite</groupId>
+        <artifactId>ignite-parent</artifactId>
+        <version>1</version>
+        <relativePath>../../parent</relativePath>
+    </parent>
+
+    <artifactId>spring-boot-ignite-autoconfigure</artifactId>
+    <version>2.9.0-SNAPSHOT</version>
```

Comment:
this version will need to be `1.0.0-SNAPSHOT`

Reply:
Why is that?
It seems the version of the module should be equal to the Ignite version.

Reply:
I agree with Saikat. If I am not mistaken, each extension should have its own release cycle and so, it is not a good idea to have the same version. Please take a look at the following thread on the dev-list:
http://apache-ignite-developers.2346864.n4.nabble.com/DISCUSS-dependencies-and-release-process-for-Ignite-Extensions-td44478.html

Reply:
So, if my extension has its own release cycle then nothing wrong to start with the 2.9.0 version? :)
Anyway, I've changed version to the `1.0.0-SNAPSHOT` just to get to the consensus here :)

JFYE: Early adopters of these modules request the release, already

Reply:
@nizhikov We are planning to start the releases with version 1.0.0 and then onwards each modules will only need to be released if they had any changes. The changes will be related to support ignite-core or other dependency modules of the future versions.